### ISSUE TITLE
Tabbed Nav – aria-orientation should be consistent

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#446]
**User Story:**
Tabbed Nav – aria-orientation should be consistent.
Current State: 
Angular and React are applying aria-orientation to different elements. 
Vanilla doesn’t appear to be working at all. 

**Notes:**
Requestor: Bob:
[WAI documentation](https://www.w3.org/TR/wai-aria-practices/#wai-aria-roles-states-and-properties-20)

**AC:**
- figure out what consistent will be for aria-orientation across frameworks
- work with PO to create updated issues for implementing to HTML, Angular and React

**Comment by ssabo1** [comment]
Will be completed with tabs issue after spark components are audited.
